speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/ConfigTests.cpp

```diff
@@ -59,7 +59,7 @@ END_TEST_COMBINATORIAL3(IndicatorTest, booleans, 2, booleans, 2, booleans, 2)
 void PControlTest1(bool p_bool1, bool p_bool2, bool p_bool3)
 {
     SDAConfig config;
-    tParticipantControl arr = {p_bool1, p_bool2, p_bool3, NULL, NULL, NULL};
+    tParticipantControl arr = {p_bool1, p_bool2, p_bool3, NULL};
```

Comment:
Participant can now be tested completely by one test instead of two seperate tests.